Automattic/wp-calypso
# CPTs: Update toggle descriptions to clarify toggle behaviour when using a theme that supports CPTs

Related to https://github.com/Automattic/vulcan/issues/614

## Proposed Changes

* This PR adds additional text to the Testimonial toggle description, on the `wordpress.com/settings/writing/` page. 
* It also updates the cases where the Portfolio toggle description text is displayed - from just WoA, to WoA and Simple (this description was modified here: https://github.com/Automattic/wp-calypso/pull/94899)

## Why are these changes being made?

* This is designed to explain why the toggles will not have an affect on sites who use themes that support Jetpack CPTs (as this will be active regardless).

## Testing Instructions

* The best way to test would be to use the Calypso Live link provided in the generated comment below. Using that, visit the following page, replacing the container-live URL if necessary, and replacing the URL at the end with that of a WoA test site, a Simple test site, and a self-hosted test site: `https://container-naughty-hypatia.calypso.live/settings/writing/example-woa-site.com`. You should see the additional text in the following cases:
  * Portfolios toggle:
    * For WoA and Simple sites
  * Testimonial toggle: 
    * For self-hosted, WoA and Simple sites
* You should not see the additional text in the following cases:
  *  Portfolios toggle:
    * For a self-hosted site

~Note: I was unable to test the Calypso live link for a WoA site (same with wp-calypso local development, with and without changes), so I suspect since the previous PR to update the portfolio toggle text there have been some other environment changes here. As this is just a text change I should be able to test live after deployment as well, and can then follow up with another PR if needed.~

Screenshot showing additional text:

<img width="710" alt="Screenshot 2024-12-13 at 15 19 16" src="https://github.com/user-attachments/assets/4a1fc9a4-7dc7-424b-9bc0-c9d7d5792352" />


## Pre-merge Checklist
